src/app/(marketing)/pricing/components/PricingCards.module.css

```
@@ -0,0 +1,414 @@
+  }
+}
+
+@media (hover: none), (pointer: coarse) {
+  .cards :global(.cl-pricingTableCard) {
+    transform: none;
+    will-change: auto;
+  }
+
+  .cards :global(.cl-pricingTableCard:hover) {
+    --card-lift: 0rem;
+    box-shadow: none;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)